src/certificate-authority.ts

```diff
@@ -111,3 +111,43 @@ async function saveCertificateAuthorityCredentials(keypath: string) {
   let key = readFile(keypath, 'utf-8');
   await currentPlatform.writeProtectedFile(rootCAKeyPath, key);
 }
+
+
+function certErrors(): string {
+  try {
+    openssl(`x509 -in ${ rootCACertPath } -noout`);
```

Comment:
I think quotes might be a good idea here.  Might generate `ENOENT` type errors (or whatever `openssl` throws) if there are spaces in the path.